label_studio/data_import/api.py

```diff
@@ -588,8 +620,9 @@ def delete(self, request, *args, **kwargs):
     name='delete',
     decorator=swagger_auto_schema(
         tags=['Import'],
-        x_fern_sdk_group_name='files',
+        x_fern_sdk_group_name=['projects', 'files'],
```

Comment:
I think these actually should not be in the projects group; the file upload API doesn't seem to have anything project specific in it

Reply:
Agree, in general, it is better to follow the endpoint design with less hierarchy, to ensure we can rearrange objects in more global names, for example, cloud storage or model connections in projects vs. workspaces etc.